.aicoe-ci.yaml

```
@@ -1,7 +1,7 @@
 check:
   - thoth-build
 build:
-  base-image: "quay.io/thoth-station/s2i-minimal-py38-notebook:v0.3.0"
+  base-image: "quay.io/thoth-station/s2i-minimal-py38-notebook:v0.5.0"
   build-stratergy: "Source"
```

```suggestion
  build-strategy: Dockerfile
  dockerfile-path: Dockerfile
```

This would allow the CI to build from Dockerfile.

---

done and thanks